File: Modul 6/Soal 5/PRAK605-2410817210010-AryaArrozzaRidhoSyaputra.py
n=int(input())

print("Matriks A")
matrix1=[]
for i in range(n):
    row=list(map(int,input().split()))
    matrix1.append(row)

print("Matriks B")
matrix2=[]
for i in range(n):
    row2=list(map(int,input().split()))
    matrix2.append(row2)

result = [[0] * n for _ in range(n)]

for i in range(n): 
    for j in range(n):
        for k in range(n):
            result[i][j] += matrix1[i][k] * matrix2[k][j]

print("Matriks AXB")
for row in result:
        print(" ".join(map(str, row)))

File: Modul 6/Soal 5/PRAK605-2410817210010-AryaArrozzaRidhoSyaputra.c
#include <stdio.h>

void KaliMatriks(int n){
    printf("Matriks A\n");
    int matrix1[n][n];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &matrix1[i][j]);
        }
    }

    printf("Matriks B\n");
    int matrix2[n][n];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &matrix2[i][j]);
        }
    }

    int result[n][n];
    printf("Matriks AXB\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            result[i][j] = 0;
            for (int k = 0; k < n; k++) {
                result[i][j] += matrix1[i][k] * matrix2[k][j];
            }
            printf("%d ", result[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int n;
    scanf("%d", &n);
    KaliMatriks(n);
    return 0;
}